### DISCUSSION TITLE
Change the display of minutes

### DISCUSSION THREAD

**Discussion by UserAA21** [kimai/kimai#2263] [Questions & Answers]
Hello, 

Is it possible to change the display of minutes?
I wish users could only select 00, 15, 30, 45 or 00, 10, 20, 30, 40, 50?
![screen01](https://user-images.githubusercontent.com/77479293/104689456-d466f380-5702-11eb-898c-1cea3348b921.png)

**Comment by kevinpapst** [comment]
Not yet possible, but with the next release the minute dropdown in the project edit/create form will have 15 minute steps, so users will see only `00, 15, 30, 45` - applies to all 3 fields (order date, start date, end date).

See #2264
